src/execution/README.md

```diff
@@ -7,3 +7,8 @@ fulfilling a GraphQL request.
 import { execute } from 'graphql/execution'; // ES6
 var GraphQLExecution = require('graphql/execution'); // CommonJS
 ```
+
+```js
+import { subscribe, createSourceEventStream } from 'graphql/execution'; // ES6
+var GraphQLSubscription = require('graphql/execution'); // CommonJS
+```
```

Comment:
We don't need two examples here.
Original one is more than enough to illustrate the point of using subfolders both as CommonJS and ESM modules.